practice/7_concurrency/task2_apod_api/template.py

```diff
@@ -1,15 +1,40 @@
-API_KEY = "YOUR_KEY"
+import os
+import requests
+import concurrent.futures 
+from datetime import datetime, timedelta
+
+API_KEY = "APIKEY"
```

Comment:
you could store the api_key in a python file which could be used as a config. Do not upload the file to GitHub:
so.. the myCfg.py file:`api_key="My Api Key"`

Then you could be as following:
```
import myCfg as cfg
API_KEY = cfg.api_key
```

Another options to store credentials are
- 3rd Secret managers
- Environment variable